res/controllers/common-hid-packet-parser.js

```diff
@@ -9,6 +9,22 @@
 this.HIDDebug = function(message) {
     console.log(`HID ${message}`);
 };
+/**
+ * creates a `DataView` from any ArrayBuffer, TypedArray
+ * or plain Array (clamped to 8-bit width).
+ *
+ * @param {number[] | ArrayBuffer | TypedArray} bufferLike Object that can be represented as a sequence of bytes
+ * @returns {DataView} dataview over the bufferLike object
+ */
+const createDataView = function(bufferLike) {
+    if (Array.isArray(bufferLike)) {
+        return new Uint8ClampedArray(bufferLike).buffer;
+    } else if (ArrayBuffer.isView(bufferLike)) {
+        return bufferLike.buffer;
+    } else {
+        return bufferLike;
+    }
+};
```

Comment:
```suggestion
const createDataView = function (bufferLike) {
    if (Array.isArray(bufferLike)) {
        return new DataView(new Uint8ClampedArray(bufferLike).buffer);
    } else {
        return new DataView(bufferLike.buffer);
    }
};
```
This function doesn't generate a DataView and my mapping crashes with type error therefore. The modified code above works at least in my mapping.

Reply:
Yep, sorry. For some reason I got it mixed up and tried to return an `ArrayBuffer` :sweat_smile:

Reply:
Fixed and force-pushed.